Source/VolunteerReporting/VolunteerReporting.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
+	"folders": [
+		{
+			"path": "."
+		},
+		{
+			"path": "/Users/einari/Projects/RedCross/cbs/Source/Infrastructure"
```

Comment:
Not sure what this does, but the path is specific to your machine, can this be made generic?

Reply:
Workspaces are brilliant in VSCode - its VSCodes answer to SLN files - just slightly more readable and more oriented around folders.  But yeah, relative paths - coming up..